roofit/roofitcore/test/TestStatistics/testLikelihoodGradientJob.cpp

```diff
@@ -488,7 +476,7 @@ TEST_P(LikelihoodGradientJob, Gaussian1DAlsoWithLikelihoodJob)
    double mu0 = mu->getVal();
    double muerr0 = mu->getError();
 
-   *values = *savedValues;
+   *values = savedValues;
```

Comment:
```suggestion
   values->assign(savedValues);
```
Please don't use `RooAbsCollection::operator=`, as it can be easily confused with copying. The "assign" function is equivalent and less ambiguous

Reply:
Done.